gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -186,7 +187,7 @@ private static void configureErrorProneOptions(
         errorProneOptions.setEnabled(true);
         errorProneOptions.setDisableWarningsInGeneratedCode(true);
         errorProneOptions.setExcludedPaths(
-                String.format("%s/(build|src/generated.*)/.*", project.getProjectDir().getPath()));
+                String.format("%s/(build|src/generated.*)/.*", Pattern.quote(project.getProjectDir().getPath())));
```

Comment:
Heh, this may produce a mix of forward and back slashes, but the jvm should handle that fine (assuming jre 1.5+!)